app/src/Components/Event/EventSummary.tsx

```diff
@@ -6,6 +6,7 @@ import EventThumbnail from './EventThumbnail'
 import EventTime from './EventTime'
 import Feather from 'react-native-vector-icons/Feather'
 import { Event } from '@/Services/modules/events'
+import { navigate } from '@/Navigators/utils'
```

Comment:
Don't think you need this as it's not being used in this file?

Reply:
Removed

Reply:
This looks like it's still there in the code?